tests/python/unittest/test_operator.py

```diff
@@ -6966,6 +6966,57 @@ def test_roi_align_autograd(sampling_ratio=0):
     test_roi_align_value(2)
     test_roi_align_autograd()
 
+@with_seed()
```

Comment:
Please indicate how many times you've run these 2 tests on your own machine in the comments section